tests/server-integration/Tag/TagTest.hs

```diff
@@ -9,6 +9,7 @@ module Tag.TagTest
   , unit_tag_tag_not_found
   , unit_tag_duplicate_tag
   , unit_tag_updates_only_entry's_modification_time
+  , addOrRemoveTag
```

Comment:
I don't think this export is needed

Reply:
But I need it for Common.hs It's used to add incorrect tag

Reply:
Ah I see, in that case let's move it to the `Utils` module. Test suite should be independent, they shouldn't import from each other.

Reply:
done